src/adapter/src/catalog.rs

```diff
@@ -2003,6 +2004,8 @@ pub enum DataSourceDesc {
     Webhook {
         /// An optional expression which is used to validate each request received by this webhook.
         validate_using: Option<MirScalarExpr>,
+        /// The cluster which this source is associated with.
+        cluster_id: StorageInstanceId,
```

Comment:
It's probably better to use `ClusterId` here which is a type alias for `StorageInstanceId`.

Reply:
Updated!